controllers/ovndbcluster_controller.go

```diff
@@ -719,7 +720,8 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	// cluster member so it can be resolved from outside cluster (edpm nodes)
 	if instance.Spec.NetworkAttachment != "" {
 		var dnsIPsList []string
-		for _, ovnPod := range podList.Items[:*(instance.Spec.Replicas)] {
+        // TODO(averdagu): change to built in min once go1.21 is used
```

Comment:
indent; please put your min code into a helper function. (if we could do without float64, just `return len(...) > Replicas? Replicas: len(...)`, that would be slightly less horrible.)